flink-formats/pom.xml

```diff
@@ -81,6 +81,7 @@ under the License.
 			<modules>
 				<module>flink-sql-orc</module>
 				<module>flink-sql-parquet</module>
+				<module>flink-sql-avro</module>
```

Comment:
Pleas also update https://flink.apache.org/downloads.html (Optional components)
And https://ci.apache.org/projects/flink/flink-docs-release-1.11/dev/table/connectors/formats/avro.html (the Download link and the mentioning of Hadoop 😔).

Reply:
The `org.apache.flink.tests.util.kafka.SQLClientKafkaITCase` should not need to download the dependencies anymore.

Reply:
BTW, I will open a PR for flink-web, once I merge this PR.